app.js

```diff
@@ -20,11 +20,11 @@ app.use('/api/bounty', bounty);
 const path = require('path')
 
 // Serve static files from the React frontend app
-app.use(express.static(path.join(__dirname, 'frontend/public')))
+app.use(express.static(path.join(__dirname, 'frontend/build')))
```

Comment:
looks like someone missing a couple semi-colons in this file. Can we add those? I know it'll still work without them, but y'know i like to know when a statement ends.